pennylane/devices/default_qubit.py

```diff
@@ -562,6 +562,7 @@ def capabilities(cls):
                 "torch": "default.qubit.torch",
                 "autograd": "default.qubit.autograd",
                 "jax": "default.qubit.jax",
+                "numba": "default.qubit.numba",
```

Comment:
Fly-by comment: how come we're adding the device as a passthru device? Passthru devices are those that allow backpropagation with a certain Machine Learning framework. Is that the case here?